src/app/users/create-user/create-user.component.html

```diff
@@ -0,0 +1,127 @@
+            <input matInput *ngIf="userForm.controls.sendPasswordToEmail.value === false" formControlName="email">
+            <mat-error *ngIf="userForm.controls.email.hasError('email')">
+              Email is <strong>invalid</strong>
+            </mat-error>
+            <mat-error *ngIf="userForm.controls.email.hasError('required')">
+              Email is <strong>required</strong>
+            </mat-error>
+          </mat-form-field>
+
+          <mat-form-field fxFlex="48%">
+            <mat-label>First Name</mat-label>
+            <input matInput required formControlName="firstname">
+            <mat-error *ngIf="userForm.controls.firstname.hasError('pattern')">
+              First Name <strong>cannot</strong> begin with a special character or number
+            </mat-error>
+            <mat-error *ngIf="userForm.controls.firstname.hasError('required')">
+              First Name is <strong>required</strong>
+            </mat-error>
+          </mat-form-field>
+
+          <mat-form-field fxFlex="48%">
+            <mat-label>Last Name</mat-label>
+            <input matInput required formControlName="lastname">
+            <mat-error *ngIf="userForm.controls.lastname.hasError('pattern')">
+              Last Name <strong>cannot</strong> begin with a special character or number
+            </mat-error>
+            <mat-error *ngIf="userForm.controls.lastname.hasError('required')">
+              Last Name is <strong>required</strong>
+            </mat-error>
+          </mat-form-field>
+
+          <div fxFlex="48%" class="password-never-expires-wrapper">
+            <mat-checkbox labelPosition="before" formControlName="passwordNeverExpires" >
+              Override password expiry policy
+            </mat-checkbox>
+          </div>
+
+          <div fxFlex="48%" class="send-password-to-email-wrapper">
+            <mat-checkbox labelPosition="before" formControlName="sendPasswordToEmail">
+              Auto generate password
+            </mat-checkbox>
+          </div>
+
+          <mat-form-field fxFlex="48%" *ngIf="userForm.controls.sendPasswordToEmail.value === false">
+            <mat-label>Password</mat-label>
+            <input matInput required type="password" formControlName="password">
+            <mat-error *ngIf="userForm.controls.password.hasError('required')">
+              Password is <strong>required</strong>
+            </mat-error>
+          </mat-form-field>
+
+          <mat-form-field fxFlex="48%" *ngIf="userForm.controls.sendPasswordToEmail.value === false">
+            <mat-label>Repeat Password</mat-label>
+            <input matInput required type="password" [errorStateMatcher]="repeatPasswordMatcher" formControlName="repeatPassword">
+            <mat-error *ngIf="userForm.hasError('passwordsDoNotMatch')">
+              Passwords must <strong>match</strong>
+            </mat-error>
```

Comment:
We also need a required validator. Implement something similar to https://github.com/openMF/web-app/blob/master/src/app/login/reset-password/reset-password.component.html (for error checking)